Commands/Process.php

```diff
@@ -37,13 +35,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $systemCheck = new SystemCheck();
         $systemCheck->checkRedisIsInstalled();
 
-        Access::getInstance()->setSuperUserAccess(false);
-        Plugin\Manager::getInstance()->setTrackerPluginsNotToLoad(array('Provider'));
+        $trackerEnvironment = new Environment('tracker');
```

Comment:
Looks like this PR will fix a couple of issues. 

> Which I think means I missed something, there should probably be a $trackerEnvironment->destroy() call after processing is done, so control goes back to the CLI environment.

Does this mean there's more work left?